### PR TITLE
Expanding the prometheus metrics docs

### PR DESCRIPTION
**Pull request by nniehoff**
This PR provides some additional optional configuration around the Prometheus metrics.  These settings provide slightly more insight into the DB and Caching.  I also added a note to cover celery metrics for now.

### CODE REVIEW

File: nautobot/docs/additional-features/prometheus-metrics.md [hunk 1]
```diff
@@ -2,7 +2,30 @@
...
+## Configuration
+
+Metrics are not exposed by default. Metric exposition can be toggled with the `METRICS_ENABLED` configuration setting which exposes metrics at the `/metrics` HTTP endpoint, e.g. `https://nautobot.local/metrics`. 
+
+In addition to the `METRICS_ENABLED` setting, database and caching metrics can also be enabled by changing the database engine and caching backend:
+
+```python
+DATABASES = {
+    "default": {
+        # Other setttings...
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/prometheus-metrics.md:14]:
```suggestion
        # Other settings...
```

File: nautobot/docs/additional-features/prometheus-metrics.md [hunk 2]
```diff
@@ -2,7 +2,30 @@
...
+        "ENGINE": "django_prometheus.db.backends.postgresql",  # use "django_prometheus.db.backends.mysql" with MySQL
+    }
+}
+
+# Other settings...
+CACHES = {
+    "default": {
+        # Other setttings...
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/prometheus-metrics.md:22]:
```suggestion
        # Other settings...
```

File: nautobot/docs/additional-features/prometheus-metrics.md [hunk 3]
```diff
@@ -2,7 +2,30 @@
 
 Nautobot supports optionally exposing native Prometheus metrics from the application. [Prometheus](https://prometheus.io/) is a popular time series metric platform used for monitoring.
 
-Nautobot exposes metrics at the `/metrics` HTTP endpoint, e.g. `https://nautobot.local/metrics`. Metric exposition can be toggled with the `METRICS_ENABLED` configuration setting. Metrics are not exposed by default.
+## Configuration
+
+Metrics are not exposed by default. Metric exposition can be toggled with the `METRICS_ENABLED` configuration setting which exposes metrics at the `/metrics` HTTP endpoint, e.g. `https://nautobot.local/metrics`. 
+
+In addition to the `METRICS_ENABLED` setting, database and caching metrics can also be enabled by changing the database engine and caching backend:
```
Comment (glennmatthews) [inline nautobot/docs/additional-features/prometheus-metrics.md:9]:
```suggestion
In addition to the `METRICS_ENABLED` setting, database and/or caching metrics can also be enabled by changing the database engine and/or caching backends from `django.db.backends` / `django_redis.cache` to `django_promethus.db.backends` / `django_promethus.cache.backends.redis`:
```

**Review by glennmatthews — COMMENTED** [review]
Thanks for continuing to drive improvements in our documentation!

**Review by glennmatthews — APPROVED** [review]

**Review by jathanism — APPROVED** [review]

**Pull request merged**